t/42-screenshots.t

```diff
@@ -85,4 +87,13 @@ subtest 'screenshots are unique' => sub {
     is $whatever[1], undef, 'no second result';
 };
 
+subtest 'no errors in database log' => sub {
+    my $prep = $schema->storage->dbh->prepare('select pg_current_logfile()');
+    $prep->execute;
+    my $db_log_file  = File::Spec->catfile($ENV{TEST_PG} =~ s/^.*host=//r, $prep->fetchrow_arrayref->[0]);
+    my $log          = Mojo::File->new($db_log_file)->slurp;
+    my $error_in_log = $log =~ /duplicate.*violates unique constraint "screenshots_filename"/;
+    isnt $error_in_log, 1, 'no unique constraint error in dabatabase logs';
```

Comment:
i would probably just have used the shorter `unlike qr/.../, $log;` here.

Reply:
https://github.com/os-autoinst/openQA/pull/2600